todo_app/app.py

```diff
@@ -1,11 +1,32 @@
 from flask import Flask
+from flask import render_template, request, redirect, url_for
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items
+from todo_app.data.session_items import add_item
+
 
 app = Flask(__name__)
-app.config.from_object(Config())
+app.config.from_object(Config)
+
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    # show all to-do items
+    items = get_items()
+    return render_template("index.html", items=items)
+
+
+@app.route('/New_List', methods=["POST"])
+def New_List():
```

Comment:
Minor, but this name feels a little odd - we're adding a new to-do item, or a new list item, but I wouldn't call it a new list. This might feel unimportant, but actually having accurate and obvious names is really valuable for making your code simpler to understand!